backend/backend.go

```diff
@@ -282,7 +287,17 @@ func (backend *Backend) CreateAndAddAccount(
 		accountAdded = true
 	case *eth.Coin:
```

Comment:
will probably get rid of this function as it does too many things at once. later though :)